go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/aws/amazon-vpc-cni-k8s v1.7.2
 	github.com/aws/aws-sdk-go v1.34.24
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/giantswarm/apiextensions/v2 v2.2.0
+	github.com/giantswarm/apiextensions/v2 v2.2.1-0.20200917110432-a3d70799ad31
```

Comment:
Once `apiextensions` is sorted we need to use the released tag here again.

Reply:
@paurosello let's get this straight before merging though.

Reply:
Updated to release 2.5.1 already